src/messages/trial.messages.js

```diff
@@ -165,7 +165,7 @@ const trialMessages = defineMessages({
   },
   startInstallTrial: {
     id: 'trial.startInstallTrial',
-    defaultMessage: 'You may need to wait up to 30 minutes to access <a href="https://cloud.redhat.com/ansible/automation-hub">automation hub</a> services after the trial subscription is activated.'
+    defaultMessage: 'You may need to wait up to 30 minutes to access <a1>automation hub</a1> services after the trial subscription is activated.'
```

Comment:
`a1` is not a HTML element. Missing a `href`,

Reply:
Maybe it is here to distinguish between regular a and element that is being replaced by something else using this function and mapping above. But I would also prefer to name it differently than a or a1.

Reply:
Don't forget cloud -> console

Reply:
```
ARCHITECTURE.md
3:This repo's source code builds a static React app to be served on https://cloud.redhat.com.

src/messages/messages.js
129:        ' collections, and the hosted services on cloud.redhat.com'
146:        'Controller infrastructure is in place, connect it to the hosted services available on cloud.redhat.com.' +
```
and update src/test/components/trial/__snapshots__/trial-pages.test.js.snap snaphot

Reply:
Updated both cloud->console and `jest -u` :)